packages/flash-swap/test/integration/hifiFlashUniswapV2/effects/uniswapV2Call.ts

```diff
@@ -1,13 +1,13 @@
-import { defaultAbiCoder } from "@ethersproject/abi";
-import { BigNumber } from "@ethersproject/bignumber";
-import { Zero } from "@ethersproject/constants";
 import { USDC, WBTC, bn, hUSDC, price } from "@hifi/helpers";
-import { expect } from "chai";
-import fp from "evm-fp";
 
+import { BigNumber } from "@ethersproject/bignumber";
 import { GodModeErc20 } from "../../../../typechain";
-import { deployGodModeErc20 } from "../../../shared/deployers";
 import { HifiFlashUniswapV2Errors } from "../../../shared/errors";
+import { Zero } from "@ethersproject/constants";
+import { defaultAbiCoder } from "@ethersproject/abi";
+import { deployGodModeErc20 } from "../../../shared/deployers";
+import { expect } from "chai";
+import fp from "evm-fp";
```

Comment:
Did you install the [sort-imports](https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports) extension for VSCode? These imports are not sorted.

Reply:
Yes, I do have the extension. Just checked again, the extension is persisting this order.

Reply:
Oh the problem is then that we have different user settings. Can you set this?

```json
  "sort-imports.default-sort-style": "module",
  "sort-imports.on-save": true,
```